jerry-core/jerryscript-port.h

```diff
@@ -140,16 +140,31 @@ bool jerry_port_get_time_zone (jerry_time_zone_t *tz_p);
  */
 double jerry_port_get_current_time (void);
 
-#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
-
-#define JERRY_PORT_ENABLE_JOBQUEUE
```

Comment:
I think it makes sense to keep a separate `JERRY_PORT_ENABLE_JOBQUEUE`.
Libraries could require the job queue, but projects may not want to include Promise.